src/tests/cucumber/features/short_tests.feature

```diff
@@ -36,10 +36,10 @@ Feature: short tests
 
   @fast @short
   Scenario: 021 Four areas - DC law
-    Given the study path is "short-tests/021 Four areas - DC law"
+    Given the study path is "021 Four areas - DC law-simplified"
     When I run antares simulator
     Then the simulation takes less than 20 seconds
     And the simulation succeeds
     And the annual system cost is
-      | EXP       | STD       | MIN       | MAX       |
-      | 7.972e+10 | 2.258e+10 | 5.613e+10 | 1.082e+11 |
\ No newline at end of file
+      | EXP       | STD         | MIN         | MAX         |
+      | 6.5397e+10| 2.97462e+09 | 6.10076e+10 | 6.84837e+10 |
```

Comment:
why?

Reply:
Slight change of results after the study was cleaned up (100 TS -> 3 TS)